clients/typescript/test/migrators/builder.test.ts

```diff
@@ -120,7 +120,7 @@ test('read migration meta data', async (t) => {
   t.deepEqual(versions, ['20230613112725_814', '20230613112735_992'])
 })
 
-test('load migration from meta data', async (t) => {
+test.skip('load migration from meta data', async (t) => {
```

Comment:
Did you intend to commit this?

Reply:
Yes, that test tries to actually connect to the local electric and fails if it's actually running and a different migration was applied. I don't like actually-connecting test in the suite, so I skipped it for now, to come back at some point

Reply:
This is causing some uncaught exceptions. We need to set a mock socket.
I think this might do the trick

```suggestion
  const electric = await electrify(
    ...
    { socketFactory: new MockSocketFactory() }
  )
```

Reply:
Applied